test_util/test_upgrade_vpc.py

```diff
@@ -104,7 +104,7 @@ def create_marathon_viplisten_app():
             "gracePeriodSeconds": 300,
             "intervalSeconds": 60,
             "timeoutSeconds": 20,
-            "maxConsecutiveFailures": 3
+            "maxConsecutiveFailures": 10
```

Comment:
Note: this no longer runs the upgrade test as of today. Rather, this change must be made here:
https://github.com/mesosphere/dcos-test-utils/blob/master/advanced_tests/test_upgrade.py